src/main/java/com/smartbear/postman/PostmanImporter.java

```diff
@@ -402,37 +414,85 @@ public static void sendAnalytics() {
     }
 
     private class PostmanRestServiceBuilder extends RestServiceBuilder {
-        public RestRequest createRestServiceFromPostman(WsdlProject paramWsdlProject,
+        public RestRequest createRestServiceFromPostman(final WsdlProject paramWsdlProject,
                                                         String uri,
                                                         HttpMethod httpMethod,
                                                         String headers) throws MalformedURLException {
-            uri = convertTemplateProperties(uri);
-            RestResource restResource = createResource(
-                    ModelCreationStrategy.REUSE_MODEL,
-                    paramWsdlProject,
-                    uri);
+            RestResource restResource;
+            RestURIParserImpl uriParser = new RestURIParserImpl(uri);
+            String endpoint = uriParser.getEndpoint();
+            String resourcePath = convertTemplateProperties(uriParser.getResourcePath());
+
+            if (endpoint.contains("{{")) {
+                restResource = createResource(
+                        ModelCreationStrategy.REUSE_MODEL,
+                        paramWsdlProject,
+                        VariableUtils.convertVariables(uriParser.getEndpoint()),
+                        resourcePath,
+                        uriParser.getResourceName());
+                VariableUtils.getListOfPostmanVariables(endpoint).forEach(name -> {
+                    if (!paramWsdlProject.hasProperty(name)) {
+                        paramWsdlProject.addProperty(name);
+                    }
```

Comment:
I think we should always do this after convertVariables() to make sure that referenced properties are added to the project properties. So maybe this can be done inside convertVariables().